src/app/pages/sign-up/sign-up.page.html

```diff
@@ -0,0 +1,58 @@
+<ion-content>
+  <div class="background-images">
```

Comment:
Let's add a card to make this it's own component so it's easy for everyone else to use